scholly/base/src/global/fakeData.js

```diff
@@ -2,8 +2,8 @@ export const FAKE_TEAM_SITE_METADATA = {
   pages: ['PUBLICATIONS', 'TEAM', 'ACHIEVEMENTS'],
   publicationOptions: { groupBy: 'Category', sortBy: 'Year' },
   template: {
-    layout: '2',
-    theme: 'light',
+    layout: '1',
```

Comment:
I think we can keep this file unchanged, as what Akshay said in other PR: https://github.com/Researchify/Researchify/pull/272#discussion_r725558292